src/celldega/pre/image_quantification.py

```diff
@@ -427,7 +399,7 @@ def calculate_stain_intensities(image_file, cell_polygons, technology_name):
 
     return cell_polygons, image_array, log_transformed_stain_intensities, bin_edges
 
-def filtering_stain_positive_cells(filtering_positive_cells_threshold, bin_edges, log_transformed_stain_intensities, cell_polygons_with_metadata, base_path, image_array, use_contrast_adjusted_image=False):
+def filtering_stain_positive_cells(filtering_positive_cells_threshold, bin_edges, log_transformed_stain_intensities, cell_polygons_with_metadata, base_path, image_array, subset_bounds, use_contrast_adjusted_image=False):
```

Comment:
This is optional but if you want to give the user a little bit more power to adjust the `vmin` and `vmax` to enhance the contrast to their liking, you can take advantage of the built-in arguments  `vmin` and `vmax` in `plt.imshow()` to do that. The `vmin` and `vmax` values can be automated using `np.quantile()` or something like that.
<img width="1471" alt="Screenshot 2025-02-24 at 2 16 50 PM" src="https://github.com/user-attachments/assets/c7b8de24-4ef1-4c09-8ed2-39f07ef95efa" />